docs/practice-engine/interaction-contracts.md

```
@@ -102,24 +102,19 @@ Question displayed
 
 **Any non-last question:**
 ```text
-[ Previous ]  [ Next ]  [ Mark for review ]
+[ Previous ]  [ Next ]  ...  [ Mark for review ]
 ```
 
 **Last question:**
 ```text
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)